src/styles/components/_tag.scss

```diff
@@ -63,4 +63,19 @@
     background-color: $accent-primary-80;
     color: $text-color-primary;
   }
+
+  &-subtle {
+    @include tag-assertive;
+    @include font-family-base-regular;
+    padding: space(sm) space(md);
+    background-color: $neutral-95;
```

Comment:
We could remove `-color` from this line and L78 to achieve the same result with less code. It's only 12 characters here, which is a drop in the bucket, but the benefits of shorthand adds up to cut down the amount of code we ship to end users.